dist/grocy-meal-plan-card.js

```diff
@@ -128,21 +129,20 @@ class MealPlanCard extends LitElement {
                       ? daily.type 
                       : ""}
                 </div>
-                <img class="pic" src="${daily.picture_url}"></img>
-            </div>
-            <div class=".info"> 
-                  ${daily.type === 'recipe' && typeof daily.recipe.description !== 'undefined' && daily.recipe.description !== null
-                  ?  (document.createElement('div').innerHTML = daily.recipe.description)
-                  : ""}
+                ${ daily.picture_url !== null ? `<img class="pic" src="${daily.picture_url}"></img>` : ""}
```

Comment:
Mind throwing a `border-radius: 10px;` on the `pic` class? The rounded corners look nicer.

Reply:
Yes that looks much better. I've set it to the same setting like the meal card.